flusher/flusher/init.py

```diff
@@ -56,3 +57,14 @@ def init(chain_id, topic, db):
 			GROUP BY consensus_address, voted;"""
     )
 
+    engine.execute(
+        """CREATE VIEW oracle_script_statistics AS
```

Comment:
Calculate on each oracle script and resolve status make more sense and limit calculate only latest (1, 3, 7 days)